rpcs3/Emu/SysCalls/Modules/cellGcmSys.cpp

```diff
@@ -324,11 +363,11 @@ s32 _cellGcmInitBody(vm::pptr<CellGcmContextData> context, u32 cmdSize, u32 ioSi
 {
 	cellGcmSys.Warning("_cellGcmInitBody(context=**0x%x, cmdSize=0x%x, ioSize=0x%x, ioAddress=0x%x)", context, cmdSize, ioSize, ioAddress);
 
-	if(!local_size && !local_addr)
+	if ((!local_size && !local_addr) || (local_size == 0 && local_addr == 0))
 	{
 		local_size = 0xf900000; // TODO: Get sdk_version in _cellGcmFunc15 and pass it to gcmGetLocalMemorySize
-		local_addr = 0xC0000000;
-		vm::falloc(0xC0000000, local_size, vm::video);
+		local_addr = 0xC0000000; // TODO: Seems like it should be 0x40301400?
```

Comment:
This is a comment from, when I was figuring out and testing some stuff in cellGcm. Afterwards I realized, that this local_addr definently shouldn't be 0x40301400, but seems like I forgot to remove it.